src/task.rs

```diff
@@ -739,6 +739,7 @@ impl<'a> Runner<'a> {
                 }
                 std::io::stdout().flush().unwrap();
             } else if let Some(cmd_time) = cmd_time {
+                name = relative_path(&name, &self.cwd);
```

Comment:
Can we move the `relative_path` call into the `job.display_name` function rather?